src/crates-io/crate-removal.md

```diff
@@ -1,23 +1,21 @@
 # Crate removal procedure
 
-> **Status 2020-01-21**: this document is incomplete and unvetted. We anticipate to
-> be receiving guidelines from Mozilla Legal in the near future. If this notice
-> is still here on 2020-03-21, please comment on [this issue][outdated-issue]
-> asking for a status update.
+If we get a DMCA takedown notice, here's what needs to happen:
 
-[outdated-issue]: https://github.com/rust-lang/crates-io-cargo-teams/issues/63
+## Contact Mozilla Legal
 
-If we get a DMCA takedown notice, here's what needs to happen:
+Before removing the crates, get in touch with Mozilla Legal and ask an opinion
```

Comment:
It would be good to mention *how* to get in touch with Mozilla Legal, without that this step is essentially ping someone else on team who already knows how (which may be intentional, but then this advice should be contact those people).

Reply:
Hmm, I don't want to include the contact details publicly... I'll think of a way to store them.

Reply:
Brought this up in the core team, hopefully we'll define a proper way to contact Mozilla Legal.

Reply:
Added a link.